eZ/Publish/Core/Search/Common/Exception/InvalidFullTextSearchString.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace eZ\Publish\Core\Search\Common\Exception;
```

Comment:
I'm not sure about namespace, but for me is too specific for `\eZ\Publish\Core\Base\Exceptions` and too generic for concrete search engine impl.